internal/test/components/components.gen.go

```diff
@@ -111,9 +111,12 @@ type AdditionalPropertiesObject3 struct {
 
 // Has anonymous field which has additional properties
 type AdditionalPropertiesObject4 struct {
-	Inner                AdditionalPropertiesObject4_Inner `json:"inner"`
-	Name                 string                            `json:"name"`
-	AdditionalProperties map[string]interface{}            `json:"-"`
+	Inner struct {
+		Name                 string                 `json:"name"`
+		AdditionalProperties map[string]interface{} `json:"-"`
+	} `json:"inner"`
```

Comment:
We shouldn't be doing this here, in the case that the object already is defined 🤔 

Looks like it may only be the `client-with-responses` that is actually not generating these correctly so I'll have a look